modules/installation-aws-add-iam-roles.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-account.adoc
+
+:_content-type: PROCEDURE
+[id="specify-an-existing-iam-role_{context}"]
+= Specifying an existing IAM role
+
+Instead of allowing the installation program to create IAM instance profiles with the default permissions, you can use the `install-config.yaml` file to specify an existing IAM role for control plane and compute instances.
+
+.Prerequisites
+
+* You have an existing `install-config.yaml` file.
+
+.Procedure
+
+. Update `controlPlane.platform.aws.iamRole` with an existing role for the control plane machines.
++
+.Sample `install-config.yaml` file with an IAM role for compute instances
+[source,yaml]
+----
+compute:
+- hyperthreading: Enabled
+  name: worker
+  platform:
+    aws:
+      rootVolume:
+        iops: 2000
+        size: 500
+        type: io1
+      iamRole: ExampleRole
+----
+. Update `compute.platform.aws.iamRole` with an existing role for the compute machines.
++
+.Sample `install-config.yaml` file with an IAM role for control plane instances
+[source,yaml]
+----
+controlPlane:
+  hyperthreading: Enabled
+  name: master
+  platform:
+    aws:
+      zones:
+      - us-iso-east-1a
```

Comment:
I suggest replacing `us-iso-east-1a`/`us-iso-east-1b` with `us-east-1a`/`us-east-1b`, since `us-iso-*` (C2S) is not a **_common_** region, WDYT?

Reply:
or remove the entire `controlPlane. platform.aws.zones` and `controlPlane. platform.aws.rootVolume`, that would be more simple:

```
controlPlane:
  hyperthreading: Enabled
  name: master
  platform:
    aws:
      iamRole: ExampleRole
```